### ISSUE TITLE
Support for Built-in and Custom Plugins Syncing to the Dashboard

### ISSUE THREAD

**Issue by Firstsawyou** [apache/apisix-dashboard#1535]
# Feature request

## Please describe your feature

After modifying the plugin schema or creating a custom plugin in Apache APISIX, load the plugin to the dashboard plugin list. Currently, the recording method in FAQ.md is to only support manually loading the modified plugin `schema` or plugin. 
(https://github.com/apache/apisix-dashboard/blob/master/docs/en/latest/FAQ.md#4-after-modifying-the-plugin-schema-or-creating-a-custom-plugin-in-apache-apisix-why-cant-i-find-it-on-the-dashboard)
Can we support loading them automatically?

## Describe the solution you'd like

If possible, I think that after modifying the plugin schema or creating custom plugins in Apache APISIX, they can be automatically loaded into the dashboard plugin list. No need to manually operate step by step.

**Comment by starsz** [comment]
Yes. We had discussed it. And wait for @LiteSun 's DISCUSS to the email.

**Comment by LiteSun** [comment]
Today I will start a discussion on the mailing list.

**Comment by nic-chen** [comment]
> Today I will start a discussion on the mailing list.

looking forward to it.

**Comment by liuxiran** [comment]
Hi @LiteSun please help to update the mail link when you have time , thanks a lot

**Comment by LiteSun** [comment]
see: https://lists.apache.org/thread.html/r46f877ff436f2124a3fb0dbe5a6a5906a4e5047488096fe38ae6ac7f%40%3Cdev.apisix.apache.org%3E

**Comment by bzp2010** [comment]
According to the project principles, the plugin will be fetched directly from the Admin API, so as soon as the custom plugin is loaded it will display correctly, so this will no longer be an issue after the next renovation is complete. ref #2981

**Comment by juzhiyuan** [comment]
Refer to https://github.com/apache/apisix-dashboard/issues/2981
